manifests/config.pp

```diff
@@ -12,6 +12,26 @@
         changes => "set tftpd_flags '\"-s ${tftp::root}\"'",
       }
     }
+    if $facts['os']['family'] =~ /^(RedHat)$/ {
+      if versioncmp($facts['os']['release']['major'], '8') == 0 {
+        systemd::unit_file { 'tftp.service':
```

Comment:
I think it would be much better to use a drop in instead that just replaces ExecStart. Something like:
```puppet
include systemd
systemd::dropin_file { 'root-directory':
  unit    => 'tftp.service',
  content => "[Service]\nExecStart=\nExecStart=/usr/sbin/in.tftpd -s ${tftp::root}\n",
}

# Deal with both camptocamp/systemd < 3
if defined(Class['systemd::systemctl::daemon_reload']) {
  Systemd::Dropin_file['root-directory'] ~> Class['systemd::systemctl::daemon_reload'] -> Service[$ftp::service]
}
```